Resources/config/services.php

```diff
@@ -79,5 +81,16 @@
             ref('router')
         ])
         ->tag('console.command')
-    ;
+        ->set('sofascore.purgatory.purger.default', DefaultPurger::class);
+
+    if (class_exists('Symfony\Component\HttpKernel\HttpCache\HttpCache')) {
```

Comment:
@zlatkoverk Services are typically defined unconditionally and then altered in the extension class, for example:

* https://github.com/doctrine/DoctrineBundle/blob/a60f8bc7721d8bdbbfd0fd648946b99822bc73ea/DependencyInjection/DoctrineExtension.php#L454

So in your case, you would just remove the definition if the `HttpCache` class doesn't exist.